From 598b98d1274cea0a331fe0ade5fbcf78d13e5e95 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 18 Mar 2020 13:19:59 +0200
Subject: [PATCH] [SW-0] gaudi: asic: remove redundant include

The stdbool.h is not used in this file - remove it.

Change-Id: I99113c9162f676c23c417123fbeb641f2d2acf68
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index e6e23e2..22f43ff 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -5,8 +5,6 @@
  * SPDX-License-Identifier:     GPL-2.0+
  */
 
-#include <stdbool.h>
-
 #include <psoc_global_conf_regs.h>
 #include <psoc_pci_pll_regs.h>
 
